README.md

```diff
@@ -82,10 +89,10 @@ We strongly recommend that you install Lightly in a dedicated virtualenv, to avo
 
 ### Lightly in Action
 
-With lightly you can use latest self-supervised learning methods in a modular
+With lightly, you can use the latest self-supervised learning methods in a modular
```

Comment:
Lightly big, no?

Reply:
We said we would keep everything OSS related lowercase

Reply:
okeydokey